disabled_apps/clock/manifest.webmanifest

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Clock",
+  "description": "Gaia Clock",
+  "lang": "en",
+  "short_name": "Clock",
+  "scope": "./",
+  "start_url": "./index.html?homescreen=1",
```

Comment:
@benfrancis we don't need it, but this let's the app detect that it was launched from the homescreen.